packages/cli/test/clone-example.test.js

```diff
@@ -17,11 +17,24 @@ const VALID_EXAMPLE = 'getting-started';
 const SANDBOX_PATH = path.resolve(__dirname, 'sandbox');
 let sandbox;
 
-describe('cloneExampleFromGitHub (SLOW)', function() {
+describe.skip('cloneExampleFromGitHub (SLOW)', function() {
   this.timeout(20000);
   before(createSandbox);
   beforeEach(resetSandbox);
 
+  /**
+   * FIXME(kjdelisle): This test will prevent any meaningful changes from
+   * landing in example repositories, since it will always fail the equality
+   * test provided when new files are added, or when existing files are
+   * removed as a part of refactor/cleanup.
+   *
+   * While I do value the idea of verifying that the example packages are
+   * being cloned properly, we can't hang that idea on validating the
+   * particular presence of any content that isn't perpetually required.
+   *
+   * I'm not sure how to update this test in a useful way without creating
```

Comment:
Maybe add a comment saying this test can be removed entirely once https://github.com/strongloop/loopback-next/issues/932 is completed as that will test a project by running it, which should be adequate verification of `cloneExampleFromGitHub`